cli/remotebazel/remotebazel.go

```diff
@@ -489,6 +489,12 @@ func streamLogs(ctx context.Context, bbClient bbspb.BuildBuddyServiceClient, inv
 	var chunks []*elpb.GetEventLogChunkResponse
 	wasLive := false
 	for {
+		select {
+		case <-ctx.Done():
+			return nil
+		default:
+		}
```

Comment:
instead of this check, should we just check for `ctx.Err() != nil` after calling GetEventLogChunk?

Reply:
Ah yeah - good call. Fixed